s29862-JuliaButkiewicz/Zadania_3/RozwiązanieZadań1.txt

```diff
@@ -21,7 +21,10 @@ B: Jak wykonasz poniższe dwa zadania?
 -wyświetl wszystkie procesy należące do użytkownika root.
 
 WYKONANIE:
-Aby wyświetlić wszystkie procesy bash : "ps aux | grep bash"
+Aby wyświetlić wszystkie procesy bash : 
+"ps aux | grep -E '[b]ash' | awk '$11 !~ /bash/'" <-- Jeśli dobrze zrozumiałam
```

Comment:
Nie. Wciąż szukasz wiersza z tekstem zamiast szukać procesu z poleceniem o podanej nazwie.
Chociaż forma z awk mi się podoba to można to zrobić inaczej, poprawnie i znacznie prościej.
Poszukaj w pomocy polecenia `ps` zamiast budować logikę do tego co już de facto jest zrobione i gotowe tylko trzeeba tego użyć.

Albo poszukaj w google.
Podpowiem, że pytania zadawane po angielsku mają większą szansę na trafienie właściwej odpowiedzi.